lib/device_detector/device.rb

```diff
@@ -0,0 +1,54 @@
+class DeviceDetector
+  class Device < Parser
+
+    def known?
+      regex_meta.any?
+    end
+
+    def name
+      ModelExtractor.new(user_agent, regex_meta).call
+    end
+
+    def device_type
```

Comment:
Why not just call the method `type`? Being in the context of `Device` it shall be clear which `type` it is.